randnum.c

```diff
@@ -0,0 +1,22 @@
+#include <stdlib.h>
```

Comment:
I'm not sure if you misunderstood and thought the range was supposed to be 1 to 11 or if you just couldn't figure out how to do the 1-10 range, but either way, your program is able to generate 0 as the number, which is very much out of range. You could have fixed this with int ans = rand() % 10 + 1; to get the 1-10 range! Aside from that, you should include comments next time.